### DISCUSSION TITLE
IPV6每过一段时间会分配2个fdf1的地址 这不算bug吗

### DISCUSSION THREAD

**Discussion by pover0k** [immortalwrt/immortalwrt#1441] [Q&A]
![image](https://github.com/user-attachments/assets/f6afb3f4-39f1-40a0-8c76-f46a4c06a349)

**Comment by Fvpder** [comment]
删除接口设置中的ULA前缀即可

**Reply by pover0k** [reply]
关了的

**Comment by libriunk** [comment]
关了之后重启就没了，如果还是存在，请排查上游

**Reply by pover0k** [reply]
24年9月还是23.05呢 这个问题随着24.10的发布已经没有了
